.kokoro/continuous/samples.cfg

```diff
@@ -0,0 +1,31 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
This is interesting, are samples expected to be hosted in this repo? Ours currently live in
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/bigtable

Reply:
This will basically remain dormant until we configure the samples build and move the samples into this repo.